cla_frontend/settings/base.py

```diff
@@ -193,8 +193,21 @@ def sentry_traces_sampler(sampling_context):
     "cla_auth.middleware.ZoneMiddleware",
     "core.middleware.Cla401Middleware",
     "csp.middleware.CSPMiddleware",
+    "django_cookies_samesite.middleware.CookiesSameSite",
+    "djangosecure.middleware.SecurityMiddleware",
 )
 
+DISABLE_SAMESITE_MIDDLEWARE = os.environ.get("DISABLE_SAMESITE_MIDDLEWARE", "False") == "True"
+
+if not DISABLE_SAMESITE_MIDDLEWARE:
+    MIDDLEWARE_CLASSES = MIDDLEWARE_CLASSES + ("django_cookies_samesite.middleware.CookiesSameSite",)
+
+# Security Settings
+SECURE_CONTENT_TYPE_NOSNIFF = os.environ.get("SECURE_CONTENT_TYPE_NOSNIFF", True) == True
```

Comment:
`os.environ.get("SECURE_CONTENT_TYPE_NOSNIFF", True)` will return a string if it a `SECURE_CONTENT_TYPE_NOSNIFF` environment variable exists and it will return the second parameter if it doesn't exists which is a boolean True.

This will fail if the environment variable exists as you will be comparing "True" to True. To fix this do as you have done on line 200